roles/openshift_logging_curator/files/actions.yaml

```diff
@@ -0,0 +1,37 @@
+# ---
+# Remember, leave a key empty if there is no value.  None will be a string,
+# not a Python "NoneType"
+#
+# Also remember that all examples have 'disable_action' set to True.  If you
+# want to use this action as a template, be sure to set this to False after
+# copying it.
+# actions:
+#   1:
+#     action: delete_indices
+#     description: >-
+#       Delete indices older than 30 days (based on index name), for .defaults
+#       and .operations indices. Ignore the error if the filter does not result in an
+#       actionable list of indices (ignore_empty_list).
+#     options:
+#       # Swallow curator.exception.NoIndices exception
+#       ignore_empty_list: True
+#       # In seconds, default is 30
+#       timeout_override: ${CURATOR_TIMEOUT}
+#       # Don't swallow any other exceptions
+#       continue_if_exception: False
+#       # Optionally disable action, useful for debugging
+#       disable_action: False
+#     # All filters are bound by logical AND
```

Comment:
What does this mean?  Should we add more comments or provide a README in the logging repo with additional info. We probably will also need to document when we exit techpreview

Reply:
we might point out that curator has docs at elastic.io